kumarankit999.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "Ankit Kumar",
+  "displayStatsPublic": true,
+  "type": "personal",
+  "bio": "Front-end Developer|Open Source |Tech enthusiast",
+  "avatar": "https://github.com/kumarankit999.png"
```

Comment:
Currently, you don't need to add an avatar. You can delete it. It will automatically fetch from your GitHub ID.
``` suggestion
 
```